PatientWebApplication/SearchMicroserviceApi/Model/OfferedMedicine.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Collections.Generic;
+
+namespace SearchMicroserviceApi.Model
+{
+    public class OfferedMedicines : Medicine
+    {
+        public double price { get; set; }
```

Comment:
Write class properties to start with upper letter and remove "this" in constructors